### ISSUE TITLE
Ice and snow by create 

### ISSUE THREAD

**Issue by albertosaurio65** [DragonsPlusMinecraft/CreateCentralKitchen#49]
Many recipes from various mods call for ice, snow, etc. Acquiring these ingredients easily without being in a frozen biome is difficult, or at least I haven't found the way, so I thought if you could create a new machine or multiblock that uses water and rotary force to generate ice or snow, maybe with the same machine, both things can be done with the difference of adding one more piece such as a crusher or not.  

If you want to make it more complex, maybe you can add a new fluid, like Freon, which needs to circulate through the machine to achieve the cooling effect. Or anything more simple. 

In conclusion, what I ask for is the possibility of making ice and snow in an easy way using the create style and water

Thanks in advance :D  
![image](https://github.com/DragonsPlusMinecraft/CreateCentralKitchen/assets/70109413/c7bf6ef3-a23a-42fc-af2c-a13caf9bfabf)
